ports/atmel-samd/usb.c

```diff
@@ -243,9 +245,16 @@ static bool cdc_enabled(void) {
 }
 
 bool usb_bytes_available(void) {
+    // Check if the buffer has data, but not enough
+    // space to hold another read.
+    if (usb_rx_count > 64) {
```

Comment:
Want to use USB_RX_BUF_SIZE here so this changes with it?

Reply:
Can do. Or, we can use the EP max packet size constant; any future use of USB HS is what would be the determination of going above 64 bytes per EP buffer. Yes?